src/components/CubeListNavbar.js

```diff
@@ -340,7 +340,9 @@ const CubeListNavbar = ({
   const handleToggleSelectEmptyModal = useCallback(() => setSelectEmptyModalOpen(false), []);
 
   const enc = encodeURIComponent;
-  const sortUrlSegment = `primary=${enc(primary)}&secondary=${enc(secondary)}&tertiary=${enc(tertiary)}`;
+  const sortUrlSegment = `primary=${enc(primary)}&secondary=${enc(secondary)}&tertiary=${enc(
```

Comment:
These parameters are used for the CSV export route, but the server code (and the corresponding `sortForCSVDownload` method) wasn't updated to handle quaternary sort.